blog.html

```diff
@@ -539,7 +538,7 @@ <h3 class="form-title">Share Your Thoughts</h3>
           <label for="commentText" class="input-label">Your Comment</label>
         </div>
         <input type="hidden" id="parentCommentId" value="">
-        <button type="submit" class="submit-button" onclick="saveComment(event)">
```

Comment:
why did you remove the onClick Js from here ?

Reply:
Maybe there is some fork problem from my end. I will refork it.